opm/parser/eclipse/EclipseState/EclipseState.cpp

```diff
@@ -43,16 +43,17 @@
 #include <opm/parser/eclipse/EclipseState/SimulationConfig/SimulationConfig.hpp>
 #include <opm/parser/eclipse/EclipseState/Tables/TableManager.hpp>
 #include <opm/parser/eclipse/Parser/ParserKeywords/M.hpp>
+#include <opm/parser/eclipse/Parser/ParseContext.hpp>
 #include <opm/parser/eclipse/Units/Dimension.hpp>
 #include <opm/parser/eclipse/Units/UnitSystem.hpp>
 #include <opm/parser/eclipse/Parser/MessageContainer.hpp>
 
 
 namespace Opm {
 
-    EclipseState::EclipseState(std::shared_ptr<const Deck> deck, const ParseContext& parseContext) :
+    EclipseState::EclipseState(std::shared_ptr<const Deck> deck, const ParseContext parseContext) :
         m_deckUnitSystem(    deck->getActiveUnitSystem() ),
-        m_parseContext(      parseContext ),
+        m_parseContext(      parseContext ), // copying the reference
```

Comment:
It's not quite a reference :> Regardless, I think it's nice that when you intend to copy something like this that the argument itself is a value (as it is now), and that sufficiently communicates whats going on.